src/css/global/icons.scss

```diff
@@ -13,180 +13,192 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-EmailNotification:before {
+.icon-Letter:before {
```

Comment:
hmm, looks like you changed all the icons, in the future its nicer to add new icons to the end so we can see exactly what was added/changed. also it means i will have to redo my own icon collection with the new order

Reply:
That shouldn't be the case, I think you may be having a caching thing. Tell me more.

Reply:
yeah thats true, clearing the cache does fix it, though that could be an issue on deployment for people. thats why i would prefer we dont change all the icon codes, only add new ones to the end

Reply:
I need to add another icon for the Copy tooltip thing and this time re-did the others so that the original codes were used for existing icons. It would be nice if we didn't have to be this careful in an icon updating process that is already a bit tacky, but for now I don't want to risk testing my contention that the caching will be busted in the webpack > production pipeline... SO we can both not worry about it now :)